v2/controllers/marketplace/meterbase_controller.go

```diff
@@ -704,13 +887,40 @@ func (r *MeterBaseReconciler) installMetricStateDeployment(
 			args,
 		),
 		manifests.CreateOrUpdateFactoryItemAction(
-			serviceMonitor,
+			serviceMonitor2,
 			func() (runtime.Object, error) {
 				return factory.MetricStateServiceMonitor()
 			},
 			args,
 		),
 	}
+
+	if r.cfg.Infrastructure.OpenshiftParsedVersion().GTE(utils.ParsedVersion460) {
```

Comment:
let's move serviceMonitor0 and serviceMonitor var def here so we know it's apart of this functionality.

Reply:
ok, moved